person.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'rental'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rental
+  attr_reader :id
+
+  def initialize(age, name: 'unknown', parent_permission: true)
+    super()
+    @id = 0
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rental = []
+  end
+
+  def correct_name
+    @name
+  end
+
+  def can_use_services?
+    return true if of_age? || parent_permission
+
+    false
+  end
+
+  def add_rental(person, date)
+    Rental.new(person, self, date)
+  end
```

Comment:
Great job @FrankMugagga 🤝

- [ ] Kindly update your `add_rental` method. Kindly replace it with the method I have left in the screenshot below.
![person-add_rentals](https://github.com/FrankMugagga/OOP-school-library/assets/63917725/e98a9cc7-9b4e-4203-89c4-080120f295fa)